src/styles.sass

```diff
@@ -204,6 +204,15 @@ body
 .italic
     font-style: italic
 
+.mat-menu-panel.download
+    max-width: none !important
+    position: absolute
+    top: 13px
+    right: 0
```

Comment:
messing with the positioning of a dynamically positioned component does not seem like a great idea. Only do this if it cannot be solved in a different way.